frontend/src/modules/viewPoint/connector.ts

```diff
@@ -8,3 +9,14 @@ export const getViewPointMetadata = async (url: string): Promise<ViewPoint['meta
     return null;
   }
 };
+
+export const getViewPointCategories = async (
+  language: string,
+): Promise<ViewPointCategories | null> => {
+  try {
+    const rawViewPointsCategories = await fetchViewPointCategories({ language });
+    return adaptViewPointsCategories(rawViewPointsCategories.results);
```

Comment:
```suggestion
    const { results } = await fetchViewPointCategories({ language });
    return adaptViewPointsCategories(results);
```

Reply:
I totally agree with your suggestion, I just copied the pattern from the others `modules` in the project to keep the code consistent so I think I'll leave it as is